packages/shared/src/components/fields/form/FormWrapper.tsx

```diff
@@ -29,34 +34,44 @@ export function FormWrapper({
   leftButtonProps = {},
   rightButtonProps = {},
   title,
+  isTitleBesideCancel,
```

Comment:
is the title beside on placed instead of cancel?

Reply:
If we make it part of the button, it looks weird when hovered and it will have a different font size if you are referring to make it as one with the Cancel button.

Reply:
No not what I ment, what I'm seeing is, we either show 
- left arrow
- left arrow and title
- cancel button

so `isTitleBesideCancel` naming think gives another sense as we show cancel button and title, or do we call the back arrow cancel?

Reply:
Right good point. It would probably be weird if we have a "Cancel" and a title. But yes, we do call the back arrow as "Cancel". I will update it to make this more straightforward.

Reply:
Let me know if the changes here make it a bit more understandable https://github.com/dailydotdev/apps/pull/3043/commits/a34311396b8460a0c369c170bb9efb6140bdeeb7

Reply:
I think still the naming is a bit confusing, since its more related to title, what about `hasHeaderTitle` or `isHeaderTitle`?

Reply:
Updated. It is indeed much clearer now with `isHeaderTitle`.